go/webhook_test.go

```diff
@@ -96,6 +97,19 @@ func TestWebhook(t *testing.T) {
 			testPayload: newTestPayload(time.Now().Add(tolerance + time.Second)),
 			expectedErr: true,
 		},
+		{
+			name:        "valid multi sig is valid",
+			testPayload: newTestPayload(time.Now()),
+			modifyPayload: func(tp *testPayload) {
+				sigs := []string{
+					"v1,Ceo5qEr07ixe2NLpvHk3FH9bwy/WavXrAFQ/9tdO6mc=",
```

Comment:
Could you maybe make the correct signature *not* be the first but rather sandwiched between two failing ones?
This will make sure we don't make mistakes like: only working when the first passes, or when the last passes.

Reply:
It was always the _last_ one but I added an invalid one to the end of all the tests to cover future edge cases